### DISCUSSION TITLE
Is there any way to overwrite the github-actions[bot] user?

### DISCUSSION THREAD

**Discussion by gugaiz** [bobheadxi/deployments#79] [Help] [answered]
When I generate a deploy, they appear as they are deployed by `github-actions[bot]`. Is there any way to overwrite this with the actual user that is doing the deployment?

**Comment by bobheadxi** [comment, marked as answer]
Unfortunately, I don't think so - the only way to change this I believe is to use a custom auth token under the desired user instead of `secrets.GITHUB_TOKEN`

Each deployment is still attached to a GitHub commit however, which will have the author
